NEWS.md

```diff
@@ -1,30 +1,43 @@
-* AppArmor: update to AppArmor 4.0.1
+* AppArmor: update to latest 4.0.2 release
+* AppArmor: enable using ABI 4.0 from host parser
+* AppArmor: fix parser lookup
 * AppArmor: support AppArmor snippet priorities
+* AppArmor: allow reading cgroup memory.max file
+* AppArmor: allow using snap-exec coming from the snapd snap when starting a confined process with jailmode
 * AppArmor prompting: add checks for prompting support, include prompting status in system key, and restart snapd if prompting flag changes
 * AppArmor prompting: include prompt prefix in AppArmor rules if prompting is supported and enabled
 * AppArmor prompting: add common types, constraints, and mappings from AppArmor permissions to abstract permissions
 * AppArmor prompting: add path pattern parsing and matching
+* AppArmor prompting: add path pattern precedence based on specificity
+* AppArmor prompting: add packages to manage outstanding request prompts and rules
+* AppArmor prompting: add snap-interfaces-requests-control interface with handler-service attribute
+* AppArmor prompting: add prompting API and notice types, which require snap-interfaces-requests-control interface
+* AppArmor prompting: feature flag can only be enabled if prompting is supported, handler service connected, and the service can be started
 * Registry views (experimental): rename from aspects to registries
-* Registry views (experimental): support reading registry views using snapctl
+* Registry views (experimental): support reading registry views and setting/unsetting registry data using snapctl
+* Registry views (experimental): fetch and refresh registry assertions as needed
 * Registry views (experimental): restrict view paths from using a number as first character and view names to storage path style patterns
 * Snap components: support installing snaps and components from files at the same time (no REST API/CLI)
 * Snap components: support downloading components related assertions from the store
-* Snap components: support installing components from the store (no REST API/CLI)
-* Snap components: support removing components (REST API, no CLI)
-* Snap components: started support for component hooks
+* Snap components: support installing components from the store
+* Snap components: support removing components individually and during snap removal
 * Snap components: support kernel modules as components
+* Snap components: support for component install, pre-refresh and post-refresh hooks
+* Snap components: initial support for building systems that contain components
 * Refresh app awareness (experimental): add data field for /v2/changes REST API to allow associating each task with affected snaps
 * Refresh app awareness (experimental): use the app name from .desktop file in notifications
 * Refresh app awareness (experimental): give snap-refresh-observe interface access to /v2/snaps/{name} endpoint
+* Improve support for nvidia drivers
```

Comment:
This may be redundant with lines 73 and 74 below, unless there is more to nvidia fixes than improvements to the steam-support and opengl interfaces.

Reply:
- steam-support removes apparmor and seccomp restrictions
- opengl interface adds apparmor allow for missed drm glob
" Improve support for nvidia drivers" relates to snap-confine mount support to share correct nvidia files/libs from host

Thanks, I will clarify the last comment

Reply:
improved comment